applications/working_with_helm_charts/configuring-custom-helm-chart-repositories.adoc

```diff
@@ -15,6 +16,7 @@ The *Developer Catalog*, in the *Developer* perspective of the web console, disp
 
 As a cluster administrator, you can add multiple Helm chart repositories, apart from the default one, and display the Helm charts from these repositories in the *Developer Catalog*.
 
+As a regular user or project member with the appropriate role-based access control (RBAC) permissions, you can add multiple namespace-scoped Helm chart repositories, apart from the default one, and display the Helm charts from these repositories in the *Developer Catalog*.
```

Comment:
Consider word change: change `apart` to `separate` since `part` has multiple meanings, `separate` might be easier for translation or to quickly understand: `...Helm chart repositories, separate from the default one, and display the Helm charts...`

Reply:
When you say the default one do you mean the cluster scope repo we add by default? Maybe be more specific about what default one means.

Reply:
Agreed with David, as there is no default namespace-scoped Helm chart repositories in cluster. It will be better if we can be specific here.

Reply:
I have addressed these comments.